cmd/kube-apiserver/app/encryption/OWNERS

```diff
@@ -0,0 +1,4 @@
+approvers:
+- sig-auth-maintainers
+reviewers:
+- sig-auth
```

Comment:
Detaching aliases from specific packages makes it difficult to easily see who has experience/expertise in an area. Some of the difficulties are discussed in https://github.com/kubernetes/test-infra/issues/1701